add DThor Swap

vefam opened the pull request:
Please approve, thanks team!

libotony commented:
@Murtagh300 Recently we have added a new field `isVeWorldSupported`, could kindly add that to your file? Thanks!

> isVeWorldSupported is required, set this to true if your dApp supports the VeWorld browser extension, enabling its use within the VeWorld mobile wallet. Your dApp will then be listed in the dApp section of the VeWorld mobile discovery.

vefam commented:
> @Murtagh300 Recently we have added a new field `isVeWorldSupported`, could kindly add that to your file? Thanks!
> 
> > isVeWorldSupported is required, set this to true if your dApp supports the VeWorld browser extension, enabling its use within the VeWorld mobile wallet. Your dApp will then be listed in the dApp section of the VeWorld mobile discovery.

I have updated, please approve!
